src/content/use-cases/image-classification/deploying.mdx

```diff
@@ -6,145 +6,131 @@ githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/use-cases/
-manner.
+To deploy an image classification model using DeepSparse Engine, pass the model in the ONNX format.
+This grants the engine the flexibility to serve any model in a framework-agnostic environment.
+
+There are two options to creating the model in ONNX format:
 
-Below we describe two possibilities to obtain the required ONNX model.
+<details>
 
-#### Exporting the onnx file from the contents of a local checkpoint
+<summary><b>1) Export the ONNX/Config Files From SparseML</b></summary>
 
 This pathway is relevant if you intend to deploy a model created using SparseML library.
-For more information refer to the appropriate integration documentation in SparseML.
 
-1. The output of the SparseML training is saved to output directory `/{save_dir}` (e.g. `/trained_model`)
-2. Depending on the chosen framework, the model files are saved to `model_path`=`/{save_dir}/{framework_name}/{model_tag}` (e.g `/trained_model/pytorch/resnet50/`)
-3. To generate an onnx model, refer to the [script for image classification ONNX export](https://github.com/neuralmagic/sparseml/blob/main/src/sparseml/pytorch/image_classification/export.py).
+After training your model with SparseML, locate the `.pth` file for the checkpoint you'd like to export and run the `SparseML` integrated export script below.
 
-Example:
 ```bash
 sparseml.image_classification.export_onnx \
     --arch-key resnet50 \
     --dataset imagenet \
     --dataset-path ~/datasets/ILSVRC2012 \
     --checkpoint-path ~/checkpoints/resnet50_checkpoint.pth
 ```
-This creates `model.onnx` file, in the parent directory of your `model_path`
+This creates `model.onnx` file.
 
-####  Directly using the SparseZoo stub
+The examples below use SparseZoo stubs, but simply pass the path to `model.onnx` in place of the stubs to use the local model.
 
-Alternatively, you can skip the process of onnx model export by downloading all the required model data directly from Neural Magic's [SparseZoo](https://sparsezoo.neuralmagic.com/).
-Example:
-```python
-from sparsezoo import Model
+</details>
 
-# you can lookup an appropriate model stub here: https://sparsezoo.neuralmagic.com/
-model_stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none"
-model = Model(model_stub)
+<details>
 
-# directly download the model data to your local directory
-model_path = model.path
+<summary><b>2) Pass a SparseZoo Stub To DeepSparse</b></summary>
 
-# the onnx model file is there, ready for deployment
-import os
-os.path.isfile(model.onnx_model.path)
->>>True
-```
+This pathway is relevant if you plan to use an off-the-shelf model from the SparseZoo. 
+
+All of DeepSparse's `Pipelines` and APIs can use a SparseZoo stub in place of a local folder. 
+The `Pipelines` use the stubs to locate and download the ONNX and config files from the SparseZoo repo.
+
+All of DeepSparse's pipelines and APIs can use a SparseZoo stub in place of a local folder. 
```

Comment:
This line appears to be duplicated from above. Is that intentional? Is there a way to edit these down to avoid repetition?